utils/src/ti_logger.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright (C) 2021 Texas Instruments Incorporated - http://www.ti.com/
```

Comment:
@s-jain1 Is this copyright date correct here as well?

Reply:
Updated